cli-plugins/apps/reference/workload-create-apply.hbs.md

```diff
@@ -1,11 +1,14 @@
-# Tanzu apps workload apply
+# tanzu apps workload apply
 
-`tanzu apps workload apply` is a command used to create and update workloads that are deployed in a cluster through a supply chain.
+`tanzu apps workload apply` is a command used to create and update workloads that are deployed in a
+cluster through a supply chain.
 
-The `tanzu apps workload apply` and `tanzu apps workload create` commands have the same behavior and flags with the following exceptions:
+The `tanzu apps workload apply` and `tanzu apps workload create` commands have the same behavior and
+flags with the following exceptions:
 
 - The `tanzu apps workload create` command fails if a workload with the same name preexists on the target cluster.
-- the `update-strategy` flag is only applicable to the `tanzu apps workload apply` command. The `update-strategy` flag is not applicable to the `tanzu apps workload create` command.
+- The `update-strategy` flag is only applicable to the `tanzu apps workload apply` command.
+- The `update-strategy` flag is not applicable to the `tanzu apps workload create` command.
 
 ## Default view
```

Comment:
I have a general comment which may apply to ALL the command examples you provide throughout these docs updates.

Providing the examples without any explanation isn't as helpful as if each were preceeded by a blurb that provides context.

Also - It may be good to start with the inner-loop and progress to the git examples as that progression makes logical and time-sequence sense.

for local-path --- "when iterating on your source code, prior to checking into git, you can push your local changes.....

for git --- "once you're committed your changes, or if you want to review/validate changes committed by someone else, you can reference a specific branch, tag, or commit in your source code repo

for image --- "if you need to deploy a pre-built container image to satisfy your use case, you can reference that via the --image flag...

etc....

Reply:
1. No, the very first part explains each "section" of the command output that's why the title refers as "default view", not sure which other way to call it.
2. The examples are related to each flag usage, for more complex interactions between flags, user should visit the "Tutorials" or "how-to-guides" sections. These are meant to be a quick guide/explanation of the individual flag usage.

Reply:
I'm not sure if I follow what you're trying to suggest here, I'm open to a further discussion whenever you get a change.